applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -43,8 +43,19 @@ enum state {
 	STATE_CONNECTED_BUSY	/**< Connected, report is generated. */
 };
 
+#define SUB_PRIO_USB		-2
```

Comment:
please remove these macros - refer to later comments

Reply:
Explained in previous comments

Reply:
removed

Reply:
please remove these priorities

Reply:
Removed